_overviews/scala3-scaladoc/blog.md

```diff
@@ -33,4 +33,33 @@ Scaladoc loads blog if the `_blog` directory exists.
 All the blogpost filenames should start with date in numeric format matching `YYYY-MM-DD`.
 Example name is `2015-10-23-dotty-compiler-bootstraps.md`.
 
+## Page metadata
 
+The blog pages in scaladoc support [Yaml Frontmatter](https://assemble.io/docs/YAML-front-matter.html) which allows you to specify different values which will be used for metadata in your page. Here are the possible fields:
+
+```
+---
+layout: <A reference to the layout page for the blog page>
+author: <Name of the author of the page>
+title: <Title of the page>
+subTitle: <Subtitle of the page>
+date: <Date of the creation of the page>, e.g. 2016-12-05
+authorImg: <Link to the author's image>
+---
+<Content of your page>
+```
+
+## Special characters for the content
+Keep in mind that the writing of your blog is done with classic markdown. So here are some useful examples:
+
+` ```<content>``` ` : Multiline code block.
```

Comment:
Could you please show how this renders in a screenshot? Also, I think it would be better to lay the content on several lines, just like an actual multi-line code block.

Reply:
Hi, thanks for your feedback, so I sent a screen on the desc but the example is no longer there following feedback from SethTissue.